frontend/assets/stylesheets/components/nav/_nav.scss

```diff
@@ -150,6 +156,59 @@ $global-nav-height: 36px;
     }
 }
 
+.nav--global--subnav {
+    background-color: $c-neutral5;
+    .nav__scroll {
+        background-color: $c-neutral6;
+    }
+}
+
+.nav__list--breadcrumbs,
+.nav__list--subnav {
+
+    .nav__link {
+        padding-top: rem($gs-baseline / 2);
+        padding-bottom: rem($gs-baseline);
+    }
+}
+
+.nav__list--breadcrumbs {
+    border-right: 2px solid $white;
+
+    .nav__link {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: rem($gs-gutter / 2);
+    }
+
+    .nav__item:after {
+        content: ">";
```

Comment:
This is really minor, but this character for the divider looks a little odd. Maybe use the same  `›` character as .com?